### ISSUE TITLE
OSG running needs two extra variables set for CVMFS

### ISSUE THREAD

**Issue by duncan-brown** [ligo-cbc/pycbc#814]
The site catalog needs to set 
```
<profile namespace="env" key="GFAL_PLUGIN_DIR">/cvmfs/oasis.opensciencegrid.org/mis/osg-wn-client/3.2/3.2.37/el6-x86_64//usr/lib64/gfal2-plugins/</profile>`
```
and
```
<profile namespace="env" key="GFAL_CONFIG_DIR">/cvmfs/oasis.opensciencegrid.org/mis/osg-wn-client/3.2/3.2.37/el6-x86_64/etc/gfal2.d/</profile>`
```

**Comment by stevereyes01** [comment]
It seems the best course of action might be to set '3.2.X' to 'current'. I'll come back to patching this when I can.

**Comment by stevereyes01** [comment]
This change needs to be made in https://github.com/ligo-cbc/pycbc/blob/master/pycbc/workflow/pegasus_files/osg-site-template.xml, is that right?

**Comment by lppekows** [comment]
Right.

**Comment by stevereyes01** [comment]
@duncan-brown They are already set in the ```osg-site-template.xml``` master version. They are set under 'current' and not '3.2.37' for ```GFAL_PLUGIN_DIR``` and ```GFAL_CONFIG_DIR``` in master. I think I'm confused about what needs to be edited. These are not set in the current pycbc-releases, like 1.3.7 or 1.3.8.

**Comment by duncan-brown** [comment]
@lppekows please can you update this with exactly the environment variables needed? We're confused about what should be ``current`` and what should be specific versions?

**Comment by lppekows** [comment]
I've put a fix in https://github.com/ligo-cbc/pycbc/pull/827 and assigned Steven to verify it.  As a reminder, this will need to be cherry-picked onto the 1.3 branch if there's going to be a 1.3.9 release.

**Comment by stevereyes01** [comment]
Should be fixed with [#827](https://github.com/ligo-cbc/pycbc/pull/827) merge.
